cmd/root.go

```diff
@@ -843,8 +578,10 @@ func updateComment(ghComment github.IssueComment, jComment jira.Comment, jIssue
 // createComment uses the ID, poster username, poster name, created at time, and body
 // of a GitHub comment to generate the body of a JIRA comment, then creates it in the
 // API.
-func createComment(ghComment github.IssueComment, jIssue jira.Issue, ghClient github.Client, jClient jira.Client) error {
-	u, _, err := makeGHRequest(func() (interface{}, *github.Response, error) {
+func createComment(config lib.Config, ghComment github.IssueComment, jIssue jira.Issue, ghClient github.Client, jClient jira.Client) error {
```

Comment:
all of these functions that are *not* directly concerned with configuring and launching the command should be outside of this file and package. this function should be in lib.